### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by gworley** [web-scrobbler/web-scrobbler#3086] [Ideas]
Not sure where to ask this question.  Hope this is the place.  Something I would like to see is the ability to cache scrobbles.  Something that last.fm does that I don't like is that after approximately 3000 scrobbles they cause you not to be able to scrobble anymore.  I understand why they do it.   last.fm Desktop scrobbler will cache the overflow until they clear they block which happens everyday @ 00:00 UTC (midnight for those who don't understand the 24 hour clock.  It would be nice if this functionality would be built into this wonderful browser extension.  I have looked at it to see if I could do write it myself but I am not really certain where to begin.

Thank you,
George

**Comment by mountainash** [comment]
Could you please rename your discussion to something like "Cache scrobbles"?

Unless you code this yourself... I double it adds much value to the extension as it's a very, very edge case.
